showcase/src/Badges.elm

```diff
@@ -9,12 +9,16 @@ import Utils exposing (goToDocsCallToAction, prettifyElmCode, story)
 stories cfg =
     storiesOf
         "Badges"
-        [ oneBadge cfg Badge.light "light"
-        , oneBadge cfg Badge.dark "dark"
-        , oneBadge cfg Badge.primary "primary"
-        , oneBadge cfg Badge.warning "warning"
-        , oneBadge cfg Badge.danger "danger"
-        , oneBadge cfg Badge.success "success"
+        [ oneBadge cfg Badge.grayLight "Gray Light"
+        , oneBadge cfg Badge.primaryLight "Primary Light"
```

Comment:
Also, this string broke the code example code.

![Screenshot 2020-07-29 at 12 17 03](https://user-images.githubusercontent.com/1559013/88788344-7dda1d00-d195-11ea-947e-df7ce8c54251.png)